### PR TITLE
Pbi 16 assessor dashboard

### PR DESCRIPTION
**Pull request by indiradr**
Merge with staging branch that fixes code smells

### CODE REVIEW

File: assessor/views.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+
+
+@api_view(['GET'])
+@require_GET
+@permission_classes([IsAuthenticated])
+def assessor_dashboard(request):
+
```
Comment (johaneschristian) [inline assessor/views.py:11]: Hi, Indi! Following my comment on Arya's pull request, I would also like to suggest that you first make sure that the accessing user is an Assessor. The permission_classes([IsAuthenticated]) only make sure that a valid Authorization header is present in the request, thus the authorization for ensuring that the request.user is an Assessor must be done manually.

File: one_day_intern/settings.py [hunk 2]
```diff
@@ -40,6 +43,8 @@
     'rest_framework',
 
     'main',
+    'users',
+    'assessor',
```
Comment (johaneschristian) [inline one_day_intern/settings.py:47]: Hi, Indi! If possible, please move the assessor functionality to be under users, such that all functionalities that supports login and registration are placed under one django app.

File: assessor/views.py (hunk 1 above)
Comment (indiradr) [reply assessor/views.py:11]: Thanks for your suggestion, Jo! I'll try to implement custom permissions.

File: one_day_intern/settings.py (hunk 2 above)
Comment (indiradr) [reply one_day_intern/settings.py:47]: Hi Jo, to clarify the assessor functionality doesn't include the assessor's login and registration. It includes the dashboard and other functionalities the assessor will be able to access. For the assessor authentication it's already in the users application. Hope I cleared things up! :)

Comment (johaneschristian) [reply one_day_intern/settings.py:47]: Noted, my mistake.

**Pull request closed without merging**